examples/gpt-2/README.md

```diff
@@ -176,12 +242,10 @@ output, output_length = decoder(
 **Ex. Use 3): Fine-tuning for conditional generation**
 
 ```python
-tgt_embed = decoder.word_embedder(truth_target[:, :-1]) + decoder.position_embedder(sequence_length=tgt_len-1)
-
 output = decoder(
     memory=source_hidden_states, 
     memory_sequence_length=src_len,
-    inputs=tgt_embed,
+    inputs=input_ids,
```

Comment:
Is this correct? Where does the decoder get access to embedders if the inputs are just `input_ids`?

Reply:
https://github.com/asyml/texar-pytorch/blob/master/texar/modules/decoders/gpt2_decoder.py#L438

Reply:
Embedders are in the module.